cmd/pics/state.go

```diff
@@ -109,11 +108,73 @@ func keyTape(t *trie.Trie, number int) error {
 	return nil
 }
 
+func stateDatabaseMap(db* bolt.DB, number int) error {
+	filename := fmt.Sprintf("state_%d.dot", number)
+	f, err := os.Create(filename)
+	if err != nil {
+		return err
+	}
+	i := 0
+	prevName := ""
+	clusterData := ""
+	visual.StartGraph(f, true)
+
+	if err := db.View(func(readTx *bolt.Tx) error {
+		return readTx.ForEach(func(name []byte, b *bolt.Bucket) error {
+			if prevName != string(name) && len(prevName) != 0 {
```

Comment:
This check is not required, because this line will be executed exactly once per bucket